macros/common/stage_external_sources.sql

```diff
@@ -6,7 +6,6 @@
     
     {% for node in source_nodes %}
         
-        {% if node.external.location %}
```

Comment:
I think removing this is the reason why Snowflake tests are now failing. I added this to avoid staging sources that are not actually external—i.e. when `external` attributes are not defined. Previously, I decided the simplest way to do that was by checking `external.location`, since that's the most reliable common attribute.

I'm guessing you removed this because azuresql doesn't _actually_ need to define `external.location`. We could sub this out for some other check, to make sure that `external` is a dict with stuff inside. Maybe `if node.external.items()`?

Reply:
`{% if node.external %}` works as empty dicts evaluate to `False`